### ISSUE TITLE
Extra `]` when autocompleting

### ISSUE THREAD

**Issue by andrsd** [dschwen/autocomplete-moose#4]
Steps to reproduce:

```
- type [
  atom enters [] with cursor in the middle
- type O (or something else, works with other letters as well ;-))
  atom suggests possible values
- pick one
  value is entered but with an extra ]
```

Atom version: 1.5.4
Autocomplete-moose version: 0.19.0

**Comment by dschwen** [comment]
Yeah, it looks like Atoms automatic closing bracket insertion is to blame (or me not taking it into account). I wonder if they added `]` insertion after I wrote the autocomplete-moose plugin. I noticed it first a few weeks ago.

**Comment by andrsd** [comment]
Probably, because your demo animation does not suffer from this...

**Comment by dschwen** [comment]
Ok, this is fixed in [v0.19.1](https://github.com/dschwen/autocomplete-moose/tree/v0.19.1) it was a logic error on my part (I had already put in code that deals with the bracket automatching, just in the opposite way)
